lib/storage/index.js

```diff
@@ -131,6 +131,18 @@ function Storage(config) {
  * var storage = gcloud.storage();
  * var albums = storage.bucket('albums');
  *
+ * //-
+ * // Make a bucket's contents publicly readable.
```

Comment:
Good catch. I think this snippet would only apply to the objects already in the bucket. Maybe I should use `albums.acl.default.add` instead?